pkg/apis/kubic/v1beta1/v1beta1_suite_test.go

```diff
@@ -33,6 +33,12 @@ var cfg *rest.Config
 var c client.Client
 
 func TestMain(m *testing.M) {
+
+	if testing.Short() {
+		log.Print("Skippig  integration tests in short mode")
+		return
```

Comment:
This `return` does not do what we expect. If you are in `testing.Short()` it will skip the whole test suite, and you just want to run this suite because unit tests could exist inside (https://github.com/kubic-project/dex-operator/blob/master/pkg/apis/kubic/v1beta1/v1beta1_suite_test.go#L38-L65)